kafka-ui-api/src/main/java/com/provectus/kafka/ui/serde/ProtobufFileRecordSerDe.java

```diff
@@ -44,10 +47,19 @@ public ProtobufFileRecordSerDe(Path protobufSchemaPath, String messageName,
   @Override
   public DeserializedKeyValue deserialize(ConsumerRecord<Bytes, Bytes> msg) {
     try {
-      return new DeserializedKeyValue(
-          msg.key() != null ? new String(msg.key().get()) : null,
-          msg.value() != null ? parse(msg.value().get()) : null
-      );
+      var builder = DeserializedKeyValue.builder();
+      if (msg.key() != null) {
+        builder.key(new String(msg.key().get()));
+        builder.keyFormat(MessageFormat.UNKNOWN);
```

Comment:
As far as I see, the key format is String. Can we add one more format,e.g. MessageFormat.STRING and use it here?

Reply:
I disagree, string is not a message format (e.g. json is also a string). We don't know the real key format here that is why I use UNKNOWN.

Reply:
I think it depends on how we are going to use this in UI. Strictly speaking we always return string to client. If we want to tell **what serializer** we used to construct this string I think we should create new enum with AVRO, PROTO, JSON_SCHEMA, STRING options. 

cc @IldarAlmakaev